api/app/src/Migrations/Version286.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version286 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Adds mirgration script';
+    }
+
+    public function up(Schema $schema): void
+    {
+        if (getenv('RUN_ONE_OFF_MIGRATIONS')) {
```

Comment:
I'd check this is doing what you expect it to be doing locally by trying it as false. I have a feeling the will resolve to true even if it's set as false because it's checking on the existence of the var and the var comes in as a string. 

You might have to convert to boolean on this line. Not 100% certain but I have had issues with this sort of thing in the past